test/wasm2js/br.2asm.js

```diff
@@ -18,6 +18,10 @@ function asmFunc(imports) {
   
  }
  
+ function $0() {
+  
+ }
```

Comment:
What is the reason for these changes?

Reply:
I have no idea, and not for lack of trying to figure it out :/

Reply:
Perhaps `wasm-reduce` can help here? It could compare the old and new outputs and find the smallest possible input that causes a difference.

Reply:
Oh, I thought this was an extra empty function that was being inserted at the beginning, but it's just the names that are being changed. The addition of function `$0` in this diff is balanced by the removal of old function `$4`. The difference in function names is just a difference between the old and new text parsers when parsing unnamed functions.